packages/neptune-extension/src/common/components/checkout-modal/CheckoutModal.tsx

```diff
@@ -75,7 +75,10 @@ const CheckoutModal: React.FC<CheckoutModalProps> = ({
 
     setLoading(true);
 
-    const content = await leaderboardClient.api.getCheckpointContent({ id: checkpointId })
+    const content = await leaderboardClient.api.getCheckpointContent({ id: checkpointId });
+
+    console.debug('content', content);
```

Comment:
I will remove it before merging.
This is added to check that even that response is not specified in swagger, we still get it.

The only downside is that we don't have any typing for it, but anyway it is notebook content.
We don't know its exact shape.